server/IdentityService/Models/User.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+using IdentityService.Utils;
+
+namespace IdentityService.Models
+{
+    public class User
+    {
+        public User()
+        {
+        }
+
+        public User(string login, string password)
+        {
+            Login = login;
+            PasswordHash = PasswordUtil.CreatePasswordHash(password);
```

Comment:
As PasswordUtil is static we could go further and make an internal(limited to project scope) extension method on string